tests/end2endtest/es_server.yml

```diff
@@ -0,0 +1,87 @@
+# Easy-server server file for pywbem end2end tests
+#
+# This file has the following format:
+#
+# vault_file: VAULT_FILE           # Path name of easy-server vault file.
+#                                  # Mandatory.
+#
+# default: NICKNAME                # Nickname of default group or server.
+#                                  # Mandatory.
+#
+# server_groups:                   # WBEM server groups.
+#                                  # Optional, default: no groups defined.
+#
+#   NICKNAME42:                    # Nickname of WBEM server group.
+#
+#     - NICKNAME1                  # List of nicknames of servers or server
+#                                  # groups in that group.
+# servers:
+#
+#   NICKNAME1:                     # Nickname of the WBEM server.
+#
+#     description: TEXT            # Short description of the WBEM server.
+#                                  # Mandatory.
+#
+#     contact_name: TEXT           # Name of technical contact for the server.
+#                                  # Optional, default: null.
+#
+#     access_via: TEXT             # Short reminder on the network/firewall/
+#                                  # proxy/vpn used to access the server.
+#                                  # Optional, default: null.
+#
+#     user_defined:
+#
+#       docker_image: ORG/IMAGE:M.N.U
+#                                  # Docker image to pull for setting up the
+#                                  # WBEM server, or null to not pull any
+#                                  # Docker image.
+#                                  # Optional, default: null.
+#
+#       docker_port_mapping:       # Port mapping for Docker image.
+#                                  # Optional, default: straight mapping.
+#
+#         image: PORT              # Port number provided by Docker image.
+#                                  # Mandatory.
+#
+#         local: PORT              # Port number used on local system.
+#                                  # Mandatory.
+#
+#       url: URL                   # URL of the WBEM server; for details see url
+#                                  # parameter of pywbem.WBEMConnection().
+#                                  # Mandatory.
+#
+#       default_namespace: NAMESPACE
+#                                  # Default CIM namespace to use in the
+#                                  # created pywbem.WBEMConnection object.
+#                                  # Optional, default: null.
+#
+#       functions:                 # List of functions implemented by the
```

Comment:
The word function confused me in particular because it conflicts with programming use of the word function.  I suggest either functionality or capability or even service.  Even "server process" might be logical.

Reply:
DONE - changed to "capabilities".